src/portable/project.rs

```diff
@@ -410,6 +410,14 @@ pub fn get_default_branch_name(version: &Specific) -> String {
     String::from("edgedb")
 }
 
+pub fn get_default_user_name(version: &Specific) -> &'static str {
+    if version.major >= 6 {
+        "admin"
+    } else {
+        "edgedb"
+    }
+}
+
```

Comment:
Could we move this to branding.rs as `BRANDING_DEFAULT_USERNAME_LEGACY` (edgedb) and `BRANDING_DEFAULT_USERNAME`? This helps with searchability of the codebase as this exact string has other purposes.